src/Impostor.Api/Utils/IDateTimeProvider.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 
-namespace Impostor.Api
+namespace Impostor.Api.Utils
```

Comment:
why move classes from the public api? they were stabilized in this location, why do they need to be moved? this is pointless breakage

Reply:
To keep things clean, it really wasn't supposed to be in root namespace, and no one used it anyway